File: semester I/Praktikum Pemrograman I/Modul1_JovanGilbertNatamasindah_2310817310002/C/PRAK110-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>
#include <math.h>

void main () {
    int ALAS = 5;
    int TINGGI = 12;
    int MIRING = sqrt(ALAS*ALAS + TINGGI*TINGGI);
    int KELILING = ALAS + TINGGI + MIRING;
    int LUAS = (ALAS * TINGGI)/2;
    
    printf("Diketahui : \n");
    printf("Alas = %d cm\n", ALAS);
    printf("Tinggi = %d cm\n\n", TINGGI);
    
    printf("jawab : \n");
    printf("Sisi A = ? %d cm\n", ALAS);
    printf("Sisi B = ? %d cm\n", MIRING);
    printf("Sisi C = ? %d cm\n", TINGGI);
    printf("Keliling = %d cm\n", KELILING );
    printf("Luas = %d cm\n", LUAS );
}

File: semester I/Praktikum Pemrograman I/Modul1_JovanGilbertNatamasindah_2310817310002/Python/PRAK110-2310817310002-JovanGilbertNatamasindah.py
import math

ALAS = 5
TINGGI = 12
MIRING = round(math.sqrt(ALAS * ALAS + TINGGI * TINGGI))
KELILING = ALAS + TINGGI + MIRING
LUAS = round((ALAS * TINGGI) / 2)

print("Diketahui : ")
print(f"Alas = {ALAS} cm")
print(f"Tinggi = {TINGGI} cm\n")

print("Jawab : ")
print(f"Sisi A = ? {ALAS} cm")
print(f"Sisi B = ? {MIRING} cm")
print(f"Sisi C = ? {TINGGI} cm")
print(f"Keliling = {KELILING} cm")
print(f"Luas = {LUAS} cm")


